### PR TITLE
- Add support for Empires game

### PR DESCRIPTION
**Pull request by Neoony**
This PR adds support for Empires game

Game website: https://www.empiresmod.com/

### CODE REVIEW

File: src/game_standalone.cpp [hunk 1]
```diff
@@ -1416,6 +1435,7 @@ void* get_get_player_by_index_fn()
     {
         case STEAM_GAME_CSS:
         case STEAM_GAME_TF2:
+        case STEAM_GAME_EMPIRES:
```
Comment (crashfort) [inline src/game_standalone.cpp:1438]: Remove, game does not have this.

File: src/game_standalone.cpp [hunk 2]
```diff
@@ -1395,6 +1413,7 @@ void* get_get_spec_target_fn()
...
```
Comment (crashfort) [inline src/game_standalone.cpp:1416]: Remove, game does not have this.

File: src/game_standalone.cpp [hunk 3]
```diff
@@ -1217,6 +1229,7 @@ void* get_local_player_ptr()
...
```
Comment (crashfort) [inline src/game_standalone.cpp:1232]: Remove, game does not have this.

File: src/game_standalone.cpp [hunk 4]
```diff
@@ -510,6 +515,7 @@ float* get_player_vel(void* player)
         }
 
         case STEAM_GAME_TF2:
+        case STEAM_GAME_EMPIRES:
```
Comment (crashfort) [inline src/game_standalone.cpp:518]: Remove, game does not have this.

File: src/game_standalone.cpp [hunk 5]
```diff
@@ -451,6 +455,7 @@ void* get_local_player()
         case STEAM_GAME_CSS:
         case STEAM_GAME_CSGO:
         case STEAM_GAME_TF2:
+        case STEAM_GAME_EMPIRES:
```
Comment (crashfort) [inline src/game_standalone.cpp:458]: Remove, game does not have this.

File: src/game_standalone.cpp [hunk 6]
```diff
@@ -432,6 +435,7 @@ void* get_player_by_idx(s32 index)
...
```
Comment (crashfort) [inline src/game_standalone.cpp:438]: Remove, game does not have this.

File: src/game_standalone.cpp [hunk 7]
```diff
@@ -413,6 +415,7 @@ s32 get_spec_target()
...
```
Comment (crashfort) [inline src/game_standalone.cpp:418]: Remove, game does not have this.

File: src/game_standalone.cpp [hunk 8]
```diff
@@ -377,6 +377,7 @@ bool can_use_velo()
...
```
Comment (crashfort) [inline src/game_standalone.cpp:380]: Remove, game does not have this.

**Review by crashfort — CHANGES_REQUESTED** [review]

**Comment by Neoony** [comment]
Removed those and retested again

**Pull request merged**